packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -171,7 +184,29 @@ class RefreshIndicator extends StatefulWidget {
     this.semanticsValue,
     this.strokeWidth = RefreshProgressIndicator.defaultStrokeWidth,
     this.triggerMode = RefreshIndicatorTriggerMode.onEdge,
-  }) : _indicatorType = _IndicatorType.adaptive;
+  })  : _indicatorType = _IndicatorType.adaptive,
+        onStatusChange = null;
+
+  /// Creates [RefreshIndicator] with no spinner and calls `onRefresh` when
+  /// successfully armed by drag event. Events can be optionally listened
+  /// by using `onStatusChange` callback
+  const RefreshIndicator.noSpinner({
```

Comment:
You'll need a comment here that links to the example. Something like this (from TextButton):
```dart
/// {@tool dartpad}
/// This sample shows various ways to configure TextButtons, from the
/// simplest default appearance to versions that don't resemble
/// Material Design at all.
///
/// ** See code in examples/api/lib/material/text_button/text_button.0.dart **
/// {@end-tool}
```

Reply:
Is this for the doc generation part?